open_humans/views.py

```diff
@@ -372,28 +334,31 @@ def get_context_data(self, **kwargs):
         project_permissions = None
         granted_permissions = None
         permissions_changed = False
+
         if 'project_id' in self.activity:
             project = DataRequestProject.objects.get(
                 pk=self.activity['project_id'])
 
             project_permissions = {
                 'share_username': project.request_username_access,
-                'share_sources': project.request_sources_access,
```

Comment:
removing this is breaking part of the rendered template (it doesn't list sources that are requested)